### PR TITLE
Fix webhook creation on arb and opt sepolia

### PR DESCRIPTION
**Pull request by joshzhang5**

### CODE REVIEW

File: src/internal/dispatch.ts [hunk 1]
```diff
@@ -62,7 +62,9 @@ export async function requestHttpWithBackoff<Req, Res>(
         throw err;
       }
       // TODO: Standardize all errors into AlchemyError
-      lastError = new Error(err.response.status + ': ' + err.response.data);
+      lastError = new Error(
```
Comment (joshzhang5) [inline src/internal/dispatch.ts:65]: errors are just coming out as [Object object] right now

File: test/integration/notify.test.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ describe('E2E integration tests', () => {
     }
   ];
 
-  const webhookUrl = 'https://temp-site.ngrok.io';
+  const webhookUrl = 'https://www.example.com';
```
Comment (joshzhang5) [inline test/integration/notify.test.ts:33]: want to give ngrok a break since we have spammed them before

File: test/integration/notify.test.ts [hunk 3]
```diff
@@ -277,14 +277,15 @@ describe('E2E integration tests', () => {
   });
 
   it('create and delete NftActivityWebhook', async () => {
+    const network = Network.ETH_SEPOLIA;
     const nftActivityWebhook = await alchemy.notify.createWebhook(
       webhookUrl,
       WebhookType.NFT_ACTIVITY,
-      { filters: nftFilters, network: Network.ETH_GOERLI }
```
Comment (joshzhang5) [inline test/integration/notify.test.ts:283]: goerli is deprecated

File: test/integration/notify.test.ts [hunk 4]
```diff
@@ -297,15 +298,16 @@ describe('E2E integration tests', () => {
     ).toEqual(0);
   });
 
-  it('create and delete NftActivityWebhook', async () => {
+  it('create and delete NftMetadataUpdateWebhook', async () => {
+    const network = Network.ETH_MAINNET;
```
Comment (joshzhang5) [inline test/integration/notify.test.ts:302]: nft metadata updates only support eth/matic mainet

**Review by andrewkw1122 — APPROVED** [review]

**Review by thebrianchen — APPROVED** [review]
thanks! can you also add a changelog entry about adding webhooks support for these networks?

**Pull request merged**